### ISSUE TITLE
Magenta color support

### ISSUE THREAD

**Issue by vandah** [IgnisDa/printr#4]
Printr currently supports these colors: blue, cyan, green, red, yellow

I am wondering whether there's any reason for leaving the pink/magenta color out? (`\e[35m`)

**Comment by IgnisDa** [comment]
Yes, support for magenta can be added easily. Would you like to make a PR for it? If not i can add it myself.
